Cait's Unity Test/Assets/BouncySquare.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BouncySquare : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    void OnCollisionEnter2D(Collision2D other)
+    {
+        var vel = GetComponent<Velocity>();
+        vel.vel *= -1;
+    }
```

Comment:
Same thing here, removed any un used function, they aren't used at all and don't have to be in the script for the script to work.
Same thing with any static variable, better to have modifiable one than a static one.
Common practice for functions that aren't used externaly, or don't need to be used outside of the script to start it with private. (With or without it does the same, it just makes it clear for anyone else.)


```csharp
    [SerializeField] private int _velocityMultiplier;

    private void OnCollisionEnter2D(Collision2D other)
    {
        var vel = GetComponent<Velocity>();
        vel.vel *= _velocityMultiplier;
    }
```